src/components/ScmRepositoryCreateEditPage/ScmRepositoryCreateEditPage.tsx

```diff
@@ -266,8 +266,8 @@ export const ScmRepositoryCreateEditPage = ({ isEditPage = false }: IScmReposito
           <>You can edit current SCM Repository attributes below.</>
         ) : (
           <>
-            You can create an SCM Repository like <label>apache/maven.git</label> or <label>git/twitter4j.git</label>. It can be
-            created with either an Internal URL or External URL.
+            You can manually create SCM Repository, for example <Label>apache/maven.git</Label>or
```

Comment:
`</Label> or`
missing space

Reply:
I removed it willingly because then formatter added {' '} after the 'or' and text looked pretty ok, but guess I can add it back.

Reply:
{' '} is just space, probably the line was too long (spaces are stripped on the end of the lines, so this is the only way to add trailing space)

Reply:
I get that but looks weird :) merging then